app/src/main/java/com/teampophory/pophory/OnBoardingViewPagerAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.teampophory.pophory
+
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.graphics.drawable.toDrawable
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.teampophory.pophory.databinding.ItemOnboardingViewpagerBinding
+
+class OnBoardingViewPagerAdapter : ListAdapter<OnBoardingData, OnBoardingViewPagerAdapter.ViewHolder>(diffUtil) {
+
+    class ViewHolder(private val binding : ItemOnboardingViewpagerBinding) : RecyclerView.ViewHolder(binding.root) {
+        fun bind(item : OnBoardingData) {
+            binding.iv.background = Color.parseColor(item.image).toDrawable()
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        return ViewHolder(
+            ItemOnboardingViewpagerBinding.inflate(
+                LayoutInflater.from(parent.context),
+                parent,
+                false
+            )
+        )
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        holder.bind(currentList[position])
+    }
+
+    companion object {
+        val diffUtil = object : DiffUtil.ItemCallback<OnBoardingData>() {
```

Comment:
이건 거의 상수로 사용해서 UPPER_SNAKE로 적는게 좋을 것 같아요
```suggestion
        val DIFF_UTIL = object : DiffUtil.ItemCallback<OnBoardingData>() {
```